kof-operator/internal/controller/utils/utils.go

```diff
@@ -68,22 +74,22 @@ func GetClusterDeploymentStub(name, namespace string) *kcmv1alpha1.ClusterDeploy
 	}
 }
 
-func HandleError(ctx context.Context, reason, message string, cd *kcmv1alpha1.ClusterDeployment, err error, logKeysAndValues ...any) {
+func HandleError(ctx context.Context, reason, message string, obj runtime.Object, err error, logKeysAndValues ...any) {
 	log := log.FromContext(ctx)
 	log.Error(err, message, logKeysAndValues...)
 
 	formattedKeysValues := make([]string, 0, len(logKeysAndValues))
 	for i, value := range logKeysAndValues {
-		if i != 0 && i%2 == 0 {
+		if i != 0 && i%2 == 1 {
```

Comment:
As we want to get "description0, k1=v2, k3=v4", we need this fix:
```suggestion
		if i%2 == 0 {
```